odoo/addons/test_read_group/models.py

```diff
@@ -65,3 +65,31 @@ class OrderLine(models.Model):
 
     order_id = fields.Many2one('test_read_group.order', ondelete='cascade')
     value = fields.Integer()
+
+
+class User(models.Model):
+    _name = 'test_read_group.res.users'
+    _description = "User"
+
+    name = fields.Char(required=True)
+    task_ids = fields.Many2many(
+        'test_read_group.project.task',
+        'test_read_group_project_task_res_users_rel',
+        'test_read_group_res_users_id',
+        'test_read_group_project_task_id',
```

Comment:
ReallyLongIdentifiersArentFancy, really.
```suggestion
        'test_read_group_task_user_rel',
        'user_id',
        'task_id',
```